packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -980,11 +976,70 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     return _getTextPositionVertical(offset, verticalOffset);
   }
 
-  /// Keeping [selection]'s [TextSelection.baseOffset] fixed, move the
-  /// [TextSelection.extentOffset] down by one line.
+  // Deletes the current uncollapsed [selection].
```

Comment:
The square brackets here won't do anything because this isn't a doc comment on a public method, so you can remove them.